port/action/resource_test.go

```diff
@@ -2192,3 +2192,73 @@ func TestRequiredApprovalAll(t *testing.T) {
 		},
 	})
 }
+
```

Comment:
In one of the other tests, please add an assertion that the sort property isn't there

Reply:
done